tests/widgets/test_menu.py

```diff
@@ -1,40 +1,54 @@
 """
 A module testing napari_imagej.widgets.menu
 """
+import os
+
 import numpy
 import pytest
+import yaml
 from napari import Viewer
 from napari.layers import Image, Layer
 from napari.viewer import current_viewer
 from qtpy.QtCore import QRunnable, Qt, QThreadPool
 from qtpy.QtGui import QPixmap
 from qtpy.QtWidgets import QApplication, QHBoxLayout, QMessageBox
 
-from napari_imagej.java import JavaClasses, running_headless
+from napari_imagej import settings
+from napari_imagej.java import running_headless
+from napari_imagej.resources import resource_path
 from napari_imagej.widgets import menu
 from napari_imagej.widgets.menu import (
     FromIJButton,
     GUIButton,
     NapariImageJMenu,
+    SettingsButton,
     ToIJButton,
 )
+from tests.utils import jc
 
 
-class JavaClassesTest(JavaClasses):
-    """
-    Here we override JavaClasses to get extra test imports
-    """
+@pytest.fixture(autouse=True)
+def clean_settings():
+    # Obtain prior user settings
+    user_path = settings.user_config_path()
 
-    @JavaClasses.blocking_import
-    def ArrayImgs(self):
-        return "net.imglib2.img.array.ArrayImgs"
+    if os.path.exists(user_path):
+        # If they existed, read in the settings and delete the file
+        with open(user_path, "r") as f:
+            existing_settings = f.read()
+        os.remove(user_path)
 
-    @JavaClasses.blocking_import
-    def ImageDisplay(self):
-        return "net.imagej.display.ImageDisplay"
+        yield
 
+        # After the test, restore the file
+        with open(user_path, "w") as f:
+            f.write(existing_settings)
+    else:
+        yield
 
-jc = JavaClassesTest()
+        # After the test, remove the file
+        if os.path.exists(user_path):
+            os.remove(user_path)
```

Comment:
Maybe we should move this up? We could make it a session scope fixture that preserves the user settings at the start and restores them at the end?

Reply:
Moved it to `conftest.py` in bb040b0